pkgs/development/tools/gojsontoyaml/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "gojsontoyaml";
+  version = "3697ded27e8cfea8e547eb082ebfbde36f1b5ee6";
+
+  src = fetchFromGitHub {
+    rev = "${version}";
+    owner = "brancz";
+    repo = "gojsontoyaml";
+    sha256 = "07sisadpfnzbylzirs5ski8wl9fl18dm7xhbv8imw6ksxq4v467a";
+  };
+
+
+  vendorSha256 = null;
+
+  meta = with stdenv.lib; {
+    description = "Simply tool to convert json to yaml written in Go";
+    homepage = "https://github.com/brancz/gojsontoyaml";
+    license = licenses.mit;
+    maintainers = [ maintainers.bryanasdev000 ];
+    platforms = platforms.linux;
```

Comment:
```suggestion
```

No need to lock this to Linux, it should work anywhere Go works.

Reply:
Ok,I left it at that because it is the only platform I have to test. If that's the way, I have some other packages to change too.
I'll do a push soon.

Reply:
Done.